src/views/homepage/components/merchandising-slots/slots/hashiconf-global-slot/hashiconf-global-slot.module.css

```diff
@@ -0,0 +1,48 @@
+.root {
+  --padding: 24px;
+  --min-height: 200px;
+
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: var(--padding);
+  min-height: var(--min-height);
+  border-radius: 6px;
```

Comment:
Nit, non-blocking: if we're being strict, I think we should leverage `CardLink` here (& in other slots) for the border-radius and shadow properties. This would also give us the right `:hover` property ☺️ But... will require that `height: 100%` workaround, I think 🤔

Reply:
I will give it a shot, initially didn't go that route because card includes padding. I will actually test it though!

Reply:
ahhhh interesting... dang it, another hidden cost of diverging from the design system 😬 i guess we'd have to override our default card padding to match the designs.... so yeah no stress here 👍 would be nice if we could have full consistency (IMO), but I think ejecting from `CardLink` rather than overriding it are kind of equivalently "meh" given the design needs here. I like the call you made 💯

Reply:
I do have a hover state in place for the cards, its just hard to see since its on dark cards.

Reply:
🤦 I see it now, oops 😬 do we need a `transition` there?

Reply:
Added transition, still pretty faint. We could bump up to `box-shadow: var(--token-surface-higher-box-shadow);` to  make it a touch more noticeable. We ran into a similar thing on dotcom homepage. The darker merchandising slots have a darker box-shadow to make the hovers look consistent.

Reply:
Updated to use CardLink.

Reply:
very much agreed on how subtle it is... i wonder if we need like a `theme="dark"` for `CardLink` or something (seems like this is something that comes up a lot, so worth encoding there). Made a subtask [on the polish task you set up](https://app.asana.com/0/1202111817660946/1202127264881981/f) (thanks for setting that up btw! 💖 )